docker-compose.yml

```diff
@@ -135,8 +135,6 @@ services:
     depends_on:
       api:
         condition: 'service_healthy'
-      relay:
```

Comment:
Right now it works without relay but maybe let's keep it for the future when client can't connect to gateway directly :)

Reply:
It is using the relay but we just don't need it to be healthy when the connection starts, it will just keep retrying the connection until the relay is available.